public/js/modMap.js

```diff
@@ -25,32 +24,26 @@ function modMap() {
   console.log(article.dataset);
   var allMods = JSON.parse(article.dataset.mods);
   var defaultColor = 'grey'
-  console.log(allMods[0]["x"]);
+  console.log('allMods: ', allMods);
 
   //var rect = draw.rect(60, 20).attr({ fill: '#189968' }).stroke({ color: '#5ECCA2',  width: 2 })
   var wide = 30;
   for (var n = 0; n < 30; n++) {
     for (var i = 0; i < 10; i++) {
-      // array[i]
+      let shape;
       color = '#189968'
       var x = 200 - n
       var y = i
-      //getValueByKey("x", data)
-      // if (article[0].["x"] == x) {
-      //      console.log(x,'found');
-      //  }
-      t = checkIfModExists(allMods, x, i)
-      //console.log(t);
-      if (t) {
+
+      const mod = getMod(allMods, x, i)
+
+      if (mod) {
         var defaultColor = color
         var op = 1
-        var id = t["_id"]
```

Comment:
@nkwsy is it `_id` or `id`? https://github.com/nkwsy/Wildmile/pull/37/files#diff-966cb6067f5126e5899afb7013f0b6adR118

Reply:
it works both ways. but _id is more correct